app/Http/Middleware/SeoMiddleware.php

```
@@ -0,0 +1,49 @@
+use Symfony\Component\HttpFoundation\Response;
+
+class SeoMiddleware
+{
+    /**
+     * Here we set the dynamic meta tags that we cannot set in the config file in seotools.php
+     *
+     * @param  Closure(Request):Response  $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        OpenGraph::addImage(asset('images/og-image.jpg')); // todo change to setImage after accept this pr https://github.com/artesaos/seotools/pull/335
```

PHPMD (2.15.0)

Avoid using static access to class '\Artesaos\SEOTools\Facades\OpenGraph' in method 'handle'. (Clean Code Rules)

(StaticAccess)